### DISCUSSION TITLE
科技爱好者周刊（第 263 期）：开源软件如何赚钱？ | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#26] [General]
# 科技爱好者周刊（第 263 期）：开源软件如何赚钱？ | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-263

**Comment by terminalqo** [comment]
突然觉得，老阮的周刊，投稿者获得了见报能分享自己所见所闻，阮老师自己获得了流量，观众们获得了精彩的知识，这不是一举三得？

**Reply by eq1024** [reply]
是的这刚好也是互联网建立的初衷
